helix-term/src/ui/completion.rs

```diff
@@ -33,11 +33,57 @@ impl menu::Item for CompletionItem {
             .into()
     }
 
-    fn format(&self, _data: &Self::Data) -> menu::Row {
+    // fn label(&self, _data: &Self::Data) -> Spans {
```

Comment:
Did you mean to remove this commented code?

Reply:
Yea, this slipped through.